include/bluetooth/att.h

```diff
@@ -54,10 +54,24 @@ extern "C" {
 #define BT_ATT_LAST_ATTRIBUTE_HANDLE            0xffff
 #define BT_ATT_LAST_ATTTRIBUTE_HANDLE __DEPRECATED_MACRO BT_ATT_LAST_ATTRIBUTE_HANDLE
 
+#if defined(CONFIG_BT_EATT)
 #if defined(CONFIG_BT_TESTING)
 int bt_eatt_disconnect_one(struct bt_conn *conn);
 #endif /* CONFIG_BT_TESTING */
 
+enum bt_att_bearer_option {
+	BT_ATT_BEARER_UNENHANCED,
+	BT_ATT_BEARER_ENHANCED,
+	BT_ATT_BEARER_ANY,
+};
+
+/* Connect EATT channels */
+int bt_eatt_connect(struct bt_conn *conn, uint8_t num_channels);
+
+/* Disconnect EATT channels */
+int bt_eatt_disconnect(struct bt_conn *conn);
```

Comment:
Should we have some way of determining if or how many EATT bearers are already connected? e.g. if there's 5 connected, then the application may not want to connect more.